tests/test-dirs/expand_node/ppx-tests.t/run.t

```diff
@@ -0,0 +1,732 @@
+Dune setup
```

Comment:
You added a lot of tests to show that the feature only triggers on attributes, maybe a bit too much ? It makes the test very inconvenient to read.

I think it is less important than for the "syntax doc" query because here we don't rely on fragile "node matching" but on the iterator and the exact attribute nodes are found. It's nice to have a couple of cases showing that it doesn't over-trigger, but dozens is a bit too much.